### DISCUSSION TITLE
Django Ninja Extras integrated into Ninja

### DISCUSSION THREAD

**Discussion by asemoon** [eadwinCode/django-ninja-extra#141] [Q&A] [answered]
Hello,
I am wondering why there is a Django ninja extras. what's stopping this very useful library not be part of original Ninja?

**Comment by eadwinCode** [comment]
@asemoon I wanted to tag the thread where this has been discussed but I am still looking for it

**Comment by eadwinCode** [comment]
But in summary, there were some differences in the pattern of setup adapted by ninja-extra that are not supported in django-ninja

**Comment by eadwinCode** [comment, marked as answer]
@asemoon https://github.com/vitalik/django-ninja/issues/15#issuecomment-1488021551
